src/main/scala/org/scalajs/cli/Scalajsld.scala

```diff
@@ -143,6 +164,16 @@ object Scalajsld {
       help("help")
         .abbr("h")
         .text("prints this usage text")
+      checkConfig { c =>
+        if (c.output.isDefined)
+          reportWarning("using a single file as output (--output) is deprecated since Scala.js 1.3.0." +
+            " Use --outputDirectory instead.")
```

Comment:
Braces are required if the conditional block spans multiple lines:

```suggestion
        if (c.output.isDefined) {
          reportWarning("using a single file as output (--output) is deprecated since Scala.js 1.3.0." +
            " Use --outputDirectory instead.")
        }
```